pages/common/git-push.md

```diff
@@ -17,3 +17,7 @@
 - Remove remote branches which don't exist locally
 
 `git push --prune {{REMOTE-NAME}}`
+
+- Publish tags
```

Comment:
I did not write "remove" examples. It was done in other PRs by others.

I just added the one related to tags. If we need to fix those examples somebody should create new PR. Again I will probably be against it, because I often google on how to remove remote branch, I always remember that it is `git push`, but do not remember the exact syntax.